crates/holochain/src/core/workflow/app_validation_workflow/get_zomes_to_invoke_tests.rs

```diff
@@ -30,6 +30,8 @@ use holochain_zome_types::Action;
 use matches::assert_matches;
 use std::sync::Arc;
 
+// Deletes of deletes are not tested, as they are rejected by sys validation.
+
```

Comment:
I would not add this comment as I feel it will get lost and forgotten and if we add these tests back in later but leave in the comment then it will cause confusion.